CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # FL Accelerator Changelog
 
+<!--
+TEMPLATE FOR MONTHLY UPDATES
+
+## <Month> <Year>
+<Short paragraph highlighting the 2-3 most important changes>
+
+### FL Experience
+<Include all changes that are worth mentioning in this or the following sections. Not everything will warrant a mention (e.g. a very minor doc change, some change to our CI/CD pipelines that will not matter to end users, etc...). Include links where relevant>
+
+### Provisioning
+
+### Documentation
+
+### Repository structure
+-->
+
+## March 2023
+
+We are changing the release process of our [FL Accelerator repository](https://github.com/Azure-Samples/azure-ml-federated-learning). We are moving from monthly releases to continuous releases. We will however keep updating this changelog once a month to highlight the most significant changes.
+
+Our major updates for this month are the addition of resources to provision various flavors of FL sandboxes, a new example for Vertical FL, and better support for encryption (data at rest, communications in Vertical FL).
```

Comment:
proposed rephrasing: "and better support for confidentiality (encryption at rest, communications in VFL)."

Reply:
done